pkgs/http_profile/README.md

```diff
@@ -1,2 +1,68 @@
-An **experimental** package that allows HTTP clients outside of the Dart SDK
-to integrate with the DevTools Network tab.
+[![pub package](https://img.shields.io/pub/v/http_profile.svg)](https://pub.dev/packages/http_profile)
+[![package publisher](https://img.shields.io/pub/publisher/http_profile.svg)](https://pub.dev/packages/http_profile/publisher)
+
+A package that allows HTTP clients outside of the Dart SDK to integrate with
+the DevTools Network tab.
+
+## Using
+
+`HttpClientRequestProfile.profile` returns an `HttpClientRequestProfile` object.
+Populating the fields of that object with information about an HTTP request and
+about the response to that request will make that information show up in the
+DevTools Network tab.
```

Comment:
[DevTools Network View](https://docs.flutter.dev/tools/devtools/network).

Reply:
Done. Should I also change "Network tab" to "Network View" on line 5?

Reply:
Yes please.

Reply:
Done